src/docs/asciidoc/reference/manage/application-access-control.adoc

```diff
@@ -0,0 +1,12 @@
+:relative-path: ../../../
+include::{docdir}/variables.adoc[]
+
+As link:#manage-organization[seen previously], an *organization* can have several team or/and several applications. So we need to have the possibility to manage the access control per application.
```

Comment:
Do not use link to anchors. It doesn't work everywhere.
You should use [cross references](https://asciidoctor.org/docs/user-manual/#xref). Here it is <<reference.adoc#manage-organization, seen previously>>

Moreover, the anchor should be manage_organization instead of manage-organization